### ISSUE TITLE
Feature Request: Chat - Show Presence

### ISSUE THREAD

**Issue by Zuline** [keybase/keybase-issues#3080]
My apologies if there is either an existing feature or an existing feature request - I can find neither.

I really like the GUI chat interface, however fundamental to the utility of such chat is being able to determine the **presence** of other parties to the chat. We've all seen this in other chat apps - a green dot for available, yellow for offline, red for busy...yadahyadah.

The real requirement is pretty simple: Show me whether the other party is online and available to chat. Without presence, I might as well send them an encrypted email and they can respond whenever they like. Without presence if I want to communicate with someone **now** then I either have to send them a chat and see if they respond or simply call them on the phone. With presence I can choose to chat if they show as online or otherwise choose to call them if it's urgent.

Presence makes the whole thing more functional.

Once again, apologies if this is already covered.

**Comment by garrettoreilly** [comment]
Just my two cents, but I'd rather not have this implemented. My expectation when I message someone is that they will respond when they can, just like with a text message or an email. I don't think many would consider changing their "availability," especially if they're using Keybase chat from their phone.

If I saw someone as "away" or "offline" I'd hesitate to send them a message because of the ambiguity of how and when will they receive the message. Will they still get a notification? Will they get notified only once they open the app? Do they have to open our conversation to see any new messages? I'd be sure to look at the details of how correspondence would work with those that are "not available," but not everyone will.

**Comment by w2ak** [comment]
I do agree with @garrettoreilly on the fact that availability is no good to me, but I know that this is only a personal opinion.

Would it no be possible to make availability configurable, but not shown by default ?

**Comment by vincentdavis** [comment]
The way I use chat it would be helpful to know if the user is online. It helps to inform me if they are in the office and available or if I should look for an alternative person or resource to address an urgent issue.
I guess some type of option would be good.

**Comment by jsutton** [comment]
I think having this be configured at a team level could be useful. I don't want everyone to know my availability, but I may want my team to know. I agree that the default should be disabled.

**Comment by ubremer1234** [comment]
I'd second this request. Doesn't need to be default enabled (to address the concerns of the folks above) ...

**Comment by Wandalen** [comment]
We would rather have it as an option
